jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb

```diff
@@ -188,11 +189,11 @@
     "# Make the GET request for Subscriptions list \n",
     "subscription = session.get(subscription_url).json()\n",
     "\n",
-    "# Print subscription data\n",
-    "print(\"{} \\n{}\\nSubscription Id: {}\\n\".format(subscription['title'], subscription['description'], subscription['id']))\n",
-    "\n",
     "# Print the subscription object\n",
-    "print(json.dumps(subscription, sort_keys=True, indent=4))"
+    "print(json.dumps(subscription, sort_keys=True, indent=4))\n",
```

Comment:
I have a style preference for `pprint.pprint(subscription)` but this is fine

Reply:
This is pre-existing code that I just wanted to fix the Jupyter notebook newline at the end -- trying to keep things focused in this PR and make no code changes if thats ok.